packages/ibm-products/src/components/NotificationsPanel/NotificationsPanel.test.js

```diff
@@ -149,28 +151,31 @@ describe('Notifications', () => {
     expect(renderedEmptyStateSvg[0]).toBeTruthy();
   });
 
-  it('should render link in notification', () => {
+  it('should render link in notification', async () => {
+    const link = {
+      text: 'View logs',
+      url: 'https://www.carbondesignsystem.com/',
+    };
     renderNotifications({
       data: [
         {
           id: 0,
           type: 'informational',
           title: 'LogRhythm connection failure',
           timestamp: new Date(),
-          link: {
-            text: 'View logs',
-            url: 'https://www.carbondesignsystem.com/',
-          },
+          link,
           onNotificationClick: () => {},
         },
       ],
     });
-    const logLink = screen.getByText(/view logs/i);
-    userEvent.click(logLink);
-    expect(logLink);
+    const logLink = screen.getByRole('link');
+    expect(logLink).toHaveTextContent(link.text);
+    expect(logLink).toHaveAttribute('href', link.url);
+    // REACT 18: We were testing clicking of a link, which caused an error to be logged
+    // Error: Not implemented: navigation (except hash changes)
```

Comment:
Agree with the change, but don’t think we need to keep this comment in here lest it lives on forever.